modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -497,11 +497,30 @@ async fn rbf_withdrawals_are_rejected() -> anyhow::Result<()> {
         sats(PEG_OUT_AMOUNT_SATS)
     );
 
-    let current_balance = client.get_balance().await;
-    assert_eq!(
-        current_balance, balance_after_normal_peg_out,
-        "Balance is {current_balance}, expected {balance_after_normal_peg_out}"
-    );
+    // to prevent flakiness, we need to retry this check
```

Comment:
Yeah, the race condition here is that just because withdrawal failed, doesn't mean the refund was already processed.